src/aap_eda/core/models/base.py

```diff
@@ -40,3 +44,43 @@ def summary_fields(self):
             "id": self.id,
             "name": self.name,
         }
+
+
+class PrimordialModel(models.Model):
```

Comment:
Where comes from this "primordial" naming? is it not a bit generic/meaningless name?
Should we not use the CreatableModel and ModifiableModel that DAB provides? 
https://github.com/ansible/django-ansible-base/blob/1cf551b7834453dcfd09de1a18b44f9538f93852/ansible_base/lib/abstract_models/common.py#L113

https://github.com/ansible/django-ansible-base/blob/1cf551b7834453dcfd09de1a18b44f9538f93852/ansible_base/lib/abstract_models/common.py#L49

Reply:
@Alex-Izquierdo it's from controller: https://github.com/ansible/awx/blob/e605883592aa996ac65eceab3abc3106ab212e14/awx/main/models/base.py#L286

Reply:
yeah, but our reference should be DAB, am I wrong?

Reply:
I think the controller's way is more suitable for us than DAB's. Using CreatableModel and ModifiableModel means we almost need to touch every model to extract created and modified fields.